code/src/layouts/app-footer/AppFooter.tsx

```diff
@@ -0,0 +1,99 @@
+import { FC } from "react";
+import { Link } from "react-router-dom";
+import Box from "@mui/material/Box";
+
+import PageWrapper from "@/layouts/app-wrapper/PageWrapper";
+import AppTypography from "@/components/app-typography/AppTypography";
+import {
+  customerSupportFooterItems,
+  locationFooterItems,
+  policyFooterItems,
+  socialsFooterItems
+} from "@/layouts/app-footer/AppFooter.constants";
+
+import "@/layouts/app-footer/AppFooter.scss";
+
+const AppFooter: FC = () => {
+  const locationItems = locationFooterItems.map((item) => (
+    <Box component="li" className="footer__list-item" key={item.label}>
+      <AppTypography
+        className="footer__list-item-link"
+        translationKey={item.label}
+        component={item.href ? Link : undefined}
+        to={item.href as string}
+      />
+    </Box>
+  ));
+
+  const socialsItems = socialsFooterItems.map((item) => (
+    <Box component="li" className="footer__socials-item" key={item.href}>
+      <a href={item.href}>{item.element}</a>
+    </Box>
```

Comment:
Think about using `AppButton` or `AppTypography` with prop `component={Link}`. Maybe it would be better solution

Reply:
Thanks for suggestion, I used `Link` there. I think simple `Link` would be enough because those elements are just icons without text or some specific styling (there are no background or borders), so no need in `AppButton` or `AppTypography`.